src/lib/crypto/krb/t_fortuna.c

```diff
@@ -102,7 +102,7 @@ head_tail_test(struct fortuna_state *st)
     }
 
     for (bit = 0; bit < 8; bit++) {
-        res = ((double)abs(len - bits[bit] * 2)) / (double)len;
+        res = ((double)(len - bits[bit] * 2)) / (double)len;
```

Comment:
clang warns that `abs()` is being called on something that is unsigned.  I'll change the types instead in update.